pkgs/development/libraries/qt-6/hooks/wrap-qt-apps-hook.sh

```diff
@@ -0,0 +1,107 @@
+if [[ -z "${__nix_wrapQtAppsHook-}" ]]; then
```

Comment:
Is this different from the Qt 5 hook? Maybe we should refer them to the same file.

Reply:
They ought to be the same, but it would be strange to reference files from qt-5 when packaging qt-6, isn't it?